specs/_features/eip7805/beacon-chain.md

```diff
@@ -192,8 +207,9 @@ def verify_and_notify_new_payload(
     execution_payload = new_payload_request.execution_payload
     parent_beacon_block_root = new_payload_request.parent_beacon_block_root
     execution_requests_list = get_execution_requests_list(new_payload_request.execution_requests)
-    # [New in EIP-7805]
-    inclusion_list_transactions = new_payload_request.inclusion_list_transactions
+    inclusion_list_transactions = (
+        new_payload_request.inclusion_list_transactions
+    )  # [New in EIP-7805]
```

Comment:
Let's revert this change. I don't like that the comment forces the line wrapping.

PS: One of my medium-term goals is to make these types of comments more consistent.